src/dos/dos_keyboard_layout.cpp

```diff
@@ -344,22 +335,21 @@ KeyboardErrorCode KeyboardLayout::ReadKeyboardFile(const char *keyboard_file_nam
 	assert(data_len < UINT8_MAX);
 	static_assert(UINT8_MAX < sizeof(read_buf), "read_buf too small");
 
-	language_codes=new char*[data_len];
-	language_code_count=0;
+	language_codes = {};
```

Comment:
`std::list` has a `clear()` method for this

Reply:
Thanks; added.